.circleci/config.yml

```diff
@@ -20,6 +20,9 @@ jobs:
       - image: circleci/node:8.11
     steps:
       - checkout
+      - run:
+        name: npm build
```

Comment:
This line and the one below need 2 more spaces, then it should work

Reply:
Thanks, fixed on master right now